project_organization.md

```diff
@@ -0,0 +1,51 @@
+# OpenTracing Project Organization
+
+[OpenTracing](http://opentracing.io) is essentially a standardization effort. There are three constituencies that care about this standard:
```

Comment:
Small nit - drop "essentially".  State what opentracing IS.  The use of ANDs are fine. 

If there are things that OT is NOT that might also be useful if theres any common confusion that needs correction.